site/templates/message.html

```diff
@@ -53,122 +56,134 @@ <h2>Send Brian a message on his Tidbyt.</h2>
             </button>
           </div>
         </div>
 
         <div class="row justify-content-center">
           <div class="col-auto" id="message-response-div"></div>
         </div>
-
       </form>
     </div>
   </div>
 </div>
 
 <div class="container">
   <div class="row justify-content-center" id="stop_button">
-
     <div class="col-auto">
       <h2>Other Tidbyt actions:</h2>
     </div>
 
     <div class="col-auto">
       <button onclick="sendStop()" type="button" class="btn btn-danger" id="stop-button">
-        <span id="stop-button-span" style="display: none" class="spinner-grow spinner-grow-sm" role="status"
-          aria-hidden="true"></span>
-        Tell Brian to stop talking.
+        <span class="spinner-grow spinner-grow-sm" role="status" aria-hidden="true"></span>
+        <span class="text">Tell Brian to stop talking.</span>
       </button>
     </div>
-
   </div>
 
-  <div class="row justify-content-center" id="response_messages"></div>
+  <div class="row justify-content-center" id="stop-response-div"></div>
 </div>
 
 <script>
-  function removeAllChildNodes(parent) {
-    while (parent.firstChild) {
-      parent.removeChild(parent.firstChild);
-    }
-  }
-
-  // toggle button
-  async function toggleButton(buttonId, spinnerSpanId, enabledText, disabledText) {
-    const btn = document.getElementById(buttonId);
-    const btnSpan = document.getElementById(spinnerSpanId);
-
-    const btnContent = btn.textContent;
-
-    if (btnContent.includes(enabledText)) {
-      btn.setAttribute("disabled", "");
-      btnSpan.removeAttribute("style");
-      const inner = btn.innerHTML;
-      const newInner = inner.replace(enabledText, disabledText);
-      btn.innerHTML = newInner;
-    } else if (btnContent.includes(disabledText)) {
-      btn.removeAttribute("disabled");
-      btnSpan.setAttribute("style", "display: none");
-      const inner = btn.innerHTML;
-      const newInner = inner.replace(disabledText, enabledText);
-      btn.innerHTML = newInner;
-    }
-  }
+  const IS_MOCKING = true;
+  const mockDelay = (seconds) => new Promise((resolve) => setTimeout(resolve, seconds * 1000));
+
+  const API_URL_ROOT = "https://api.brianwtracey.com";
+  const apiFetch = async (path, options) => await fetch(`${API_URL_ROOT}/${path}`, options);
+  const api = {
+    stop: async () => {
+      if (IS_MOCKING) {
+        await mockDelay(2);
+        return "Thanks for telling Brian to stop talking.";
+      }
+
+      const response = await apiFetch("stop");
+      return await response.json();
+    },
+    message: async (body) => {
+      if (IS_MOCKING) {
+        await mockDelay(2);
+        return { status: "success" };
+      }
+
+      const response = await apiFetch("message", {
+        method: "post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(body),
+      });
+      return await response.json();
+    },
+  };
```

Comment:
With API stuff, I try to encapsulate that into its own section. Usually, I'll do this in a separate file but your case is relatively simple, plus I wasn't sure how your hosting system would grab files.

While the constant of `API_URL_ROOT` isn't necessary, it's nice to call it out to make it clear what API is being worked with.